modules/python/src2/cv2_convert.cpp

```diff
@@ -133,9 +133,10 @@ bool pyopencv_to(PyObject* o, Mat& m, const ArgInfo& info)
                typenum == NPY_BYTE ? CV_8S :
                typenum == NPY_USHORT ? CV_16U :
                typenum == NPY_SHORT ? CV_16S :
-               typenum == NPY_INT ? CV_32S :
```

Comment:
OpenCV interprets INT as int32, not int64 as it's done everywhere in Python. The patch is breaking change for many users.